cypress/e2e/DoenetML/tagSpecific/angle.cy.js

```diff
@@ -2244,4 +2244,50 @@ describe('Angle Tag Tests', function () {
 
   })
 
+  it('emphasize right angle', () => {
+    cy.window().then(async (win) => {
+      win.postMessage({
+        doenetML: `
+  <graph>
+    <angle name="a1" through="(-5,0) (0,0) (0,-5)" />
+    <angle name="a2" through="(5,0) (0,0) (0,-5)" emphasizeRightAngle="false" />
+    <angle name="a3" through="(5,0) (0,0) (0,5)" emphasizeRightAngle="$bi3" />
+  </graph>
+
+  <p>Emphasize right angle 1: <booleanInput name="bi1" bindValueTo="$a1.emphasizeRightAngle" /></p>
+  <p>Emphasize right angle 2: <booleanInput name="bi2" bindValueTo="$a2.emphasizeRightAngle" /></p>
+  <p>Emphasize right angle 3: <booleanInput name="bi3" /></p>
+
+  <p name="emphasize">Emphasize right angle: $a1.emphasizeRightAngle, $a2.emphasizeRightAngle, $a3.emphasizeRightAngle</p>
+  
+  `}, "*");
+    });
+
+    // TODO: How to check renderer itself?
```

Comment:
Open up a github issue or asana task to track this maybe?

Reply:
I wouldn't open it for this line in particular.  It's a bigger issue; none of the jsxgraph renderers have any test coverage, as I haven't figured out a simple way to test them.  I'm pretty sure that I couldn't get cypress to drag a point or otherwise interact with the graph.  I don't know how to test what jsxgraph actually puts in the DOM without depending on its internals (not that I actually put in any effort).  So the TODO was more aspirational than a short term plan.

Reply:
Created follow up task - https://github.com/Doenet/DoenetTools/issues/1941